docs/tutorial_supervised.rst

```diff
@@ -196,8 +198,8 @@ To train the dialogue policy from the command line, run
 ^^^^^^^^^^^^^^^^
 
 Now we're going to glue some pieces together to create an actual bot.
-We instantiate an ``Agent``, which owns our NLU ``Interpreter``, our
-trained ``Policy``, and a ``Domain``.
+We instantiate an ``Agent``, which owns our trained ``Policy`` and a ``Domain`` from ``models/dialogue``,
```

Comment:
trained ``Policy``, a ``Domain`` (remove the and)

Reply:
are you sure? sounds strange for me, without and

Reply:
It's a list of three things, so yeah I'd only put an "and" before the last one.